apps/api/setup.cfg

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 88
```

Comment:
Question: I'm pretty sure I asked this already, but why 88 instead of [79](https://peps.python.org/pep-0008/#maximum-line-length) or 80? It does say we can increase it up to 99 though.

Reply:
This is in accordance with the [line length in Black's code style](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#labels-line-length).